src/style.css

```diff
@@ -1,3 +1,85 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  margin: 0;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  width: 100%;
+  align-items: center;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav_item {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  margin: 0 0 0 20px;
+}
+
+.nav_item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav_link {
+  text-decoration: none;
+  display: flex;
+  text-transform: uppercase;
+  color: #000;
+  font-size: 12px;
+  white-space: nowrap;
+  font-weight: 500;
+  line-height: 60px;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  font-style: normal;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+/* [data-qa='hover'] {
+  color: #00acdc;
+} */
+ 
+:hover {
+  color: #00acdc;
+}
```

Comment:
Your `:hover` pseudo-class will apply to all hovered elements. It's better to specify which elements should change color on hover. The way it's written now, all elements will change their color to `#00acdc` when hovered. This can lead to unexpected results if you add more interactive elements to your page in the future.